Source/WebGPU/WebGPU/Device.mm

```diff
@@ -626,7 +627,7 @@ static void setOwnerWithIdentity(id<MTLResourceSPI> resource, auto webProcessID)
         return nil;
 
     if (!m_dispatchCallBuffer) {
-        m_dispatchCallBuffer = [m_device newBufferWithLength:sizeof(MTLDispatchThreadgroupsIndirectArguments) options:MTLResourceStorageModePrivate];
+        m_dispatchCallBuffer = safeCreateBuffer(sizeof(MTLDispatchThreadgroupsIndirectArguments));
```

Comment:
switched from Private -> default storage mode to avoid placing occlusion query buffers in their own heap